internal/cmdutil/multivalue.go

```diff
@@ -0,0 +1,47 @@
+package cmdutil
+
+import (
+	"strings"
+
+	"github.com/gobwas/glob"
+)
+
+// MultiValue can be used as a flag.Var type to support comma-separated lists
+// of values on the command line. The [MultiValue.ResolveArgValues] method can
+// be used to validate the values and resolve globs.
+type MultiValue []string
+
+func (mv *MultiValue) String() string {
+	return strings.Join(*mv, ", ")
+}
+
+func (mv *MultiValue) Set(v string) error {
+	split := strings.Split(v, ",")
+	*mv = split
```

Comment:
Genuine question, what is the benefit of splitting this into two lines and not assigning directly?

Reply:
I don't think there's a difference... might've been left-over from when I was first writing it and it just stayed there.

Reply:
Squished.